commands/peers/addpeer.go

```diff
@@ -39,37 +39,36 @@ func addPeerHandler(w http.ResponseWriter, r *http.Request) {
 
 	//FIXME: In the correct add process, the peer being probed would add it's details to the store once it's been validated. The code below is just a temporary stand-in to show how the API's would work
 
```

Comment:
imo above comment can be remove now. "//FIXME: In the correct add process, the peer being probed would add it's details to the store once it's been validated. The code below is just a temporary stand-in to show how the API's would work "